modules/claims_api/lib/claims_api/claim_logger.rb

```diff
@@ -47,6 +47,12 @@ def self.format_msg(tag, **params)
         msg.append("has ptcpnt_id: #{params[:ptcpnt_id]}") if params[:ptcpnt_id].present?
         msg.append("has birls_id: #{params[:birls_id]}") if params[:birls_id].present?
         msg.append("MPI Response OK: #{params[:mpi_res_ok]}") if params[:mpi_res_ok].present?
+      when 'multiple_ids'
+        msg.append("RID: #{params[:rid]}") if params[:rid].present?
```

Comment:
Request ID, I'm not sure if it's actually needed, I just moved it above the switch so anything can use it (since it's not really specific to any logging "type". That being said, I wonder if since they all have an `if`, if for [API-25019](https://vajira.max.gov/browse/API-25019) it'd make sense to remove the `case`?